pyproject.toml

```diff
@@ -210,7 +210,6 @@ dependencies = [
 
 [tool.black]
 line-length = 100
-target-version = ["py37"]  # 3.7
```

Comment:
Shouldn't we target 3.8 explicitly?

Reply:
No, they now have a feature to infer it from requires-python